src/core/tests/pass/serialization/serialize.cpp

```diff
@@ -56,6 +77,25 @@ class SerializationTest : public ov::test::TestsCommon, public testing::WithPara
     }
 };
 
+TEST(SerializationTest, WriteInReadOnly) {
+    // Create read-only files
+    ASSERT_TRUE(createReadOnlyFile(test_xml_path));
+    ASSERT_TRUE(createReadOnlyFile(test_bin_path));
```

Comment:
```suggestion
TEST_P(SerializationTest, WriteInReadOnly) {
    // Create read-only files
    ASSERT_TRUE(createReadOnlyFile((m_out_xml_path));
    ASSERT_TRUE(createReadOnlyFile(m_out_bin_path));
```

Update these variables in other places in this test